Packs/Troubleshoot/Scripts/TroubleshootTestInstance/TroubleshootTestInstance.py

```diff
@@ -36,11 +36,12 @@ def save_configuration(arguments: dict) -> List[str]:
     return get_errors(res)
 
 
-def execute_test_module(arguments: dict) -> List[str]:
+def execute_test_module_legacy(arguments: dict) -> List[str]:
```

Comment:
i think its a confusing name as this is no legacy functionality, infact it is the standard test module
i would keep the name for this and rename the new func to mention it is `debug`